src/backend/libs/efiling-diligen-client/src/main/java/ca/bc/gov/open/efilingdiligenclient/diligen/DiligenServiceImpl.java

```diff
@@ -74,6 +81,28 @@ public BigDecimal postDocument(String documentType, MultipartFile file) {
 
     }
 
+    @Override
+    public Object getDocumentDetails(BigDecimal documentId) {
+
+        logger.debug("getting document details from diligen");
+
+        documentsApi.getApiClient().setBearerToken(diligenAuthService.getDiligenJWT(diligenProperties.getUsername(), diligenProperties.getPassword()));
+
+        try {
+            InlineResponse2003 result = documentsApi.apiDocumentsFileIdDetailsGet(documentId.intValue());
```

Comment:
should we return this result ?
We will probably use the rest of the information for processing...

Reply:
I mapped a new object for return